### ISSUE TITLE
Error when connect microphone...

### ISSUE THREAD

**Issue by OleksandrBorodatyi** [michalstocki/FlashWavRecorder#70]
When you click record with the disconnected microphone. It will write in the log that no_microphone_found. But when you connect it and try record again. It will throw exceptions. Could you please fix it?

**Comment by michalstocki** [comment]
What browser (and version) do you use? I think this issue may be relative to #64. Unfortunatelly I have currently no time to work on FWR. However, thank you for reporting this issue.

**Comment by OleksandrBorodatyi** [comment]
Yes I think partly it is related to #64. But this scenario is little bit another. So there no hope that you will able to fix it?

**Comment by michalstocki** [comment]
Unfortunately I won't be a to fix it in the foreseeable future, so pull requests are welcome.
